### ISSUE TITLE
Update Data Plans table to include NASPO 

### ISSUE THREAD

**Issue by mrose914** [cal-itp/mobility-marketplace#381]
We have a NASPO contract for FirstNet for connectivity. We would like to update the Contracts Page on MobiMart to align with the CALACT conference and announcement next week, 11/3. 

See this doc for proposed options: https://docs.google.com/document/d/1Zn601IjD2J9pVPJuCaBsLHAMVvbBR5iVNH58QMI7tSM/edit

**Comment by mlgilbert18** [comment]
EOD on wednesday is great for the timeline discussed. Will add link to the google slides here once ready - hopefully within the hour

**Comment by mlgilbert18** [comment]
Here's the latest and greatest [doc](https://docs.google.com/presentation/d/1g04kFULkxpZCTDPsBKIQoOCUJqECPJrDebe5jizVU0g/edit?usp=sharing). Erika is reviewing one last time in the morning but should be good to go.

**Comment by mlgilbert18** [comment]
Erika has approved! doc is good to go

**Comment by thekaveman** [comment]
@mlgilbert18 @mrose914 thanks! Do we want to post a link to the Google Slides? Or convert to PDF?

**Comment by mlgilbert18** [comment]
I'd say convert to PDF? but I defer to @mrose914

**Comment by thekaveman** [comment]
@mrose914 sounds good, I'll convert the slides to PDF. 

We'll host it right here in the repository -- assuming the file isn't too large.

**Comment by thekaveman** [comment]
🛑 ON HOLD - do not merge the PR until 11/2 EOD 🛑
